src/lib/y2storage/storage_manager.rb

```diff
@@ -147,6 +148,13 @@ def commit
       storage.commit
     end
 
+    # Disk analyzer used to analyze the initial devicegraph
+    #
+    # @return [DiskAnalyzer]
+    def disk_analyzer
```

Comment:
I would prefer something like `probed_disk_analyzer`, to make the connection with `probed` explicit.